internal/blockchain/chain.go

```diff
@@ -467,7 +469,7 @@ func (b *BlockChain) ChainWork(hash *chainhash.Hash) (*big.Int, error) {
 		return nil, unknownBlockError(hash)
 	}
 
-	return node.workSum, nil
+	return node.workSum.ToBig(), nil
```

Comment:
Now that the blockchain package is internal would it not be better to modify the function signature to return a uint256 instead to avoid the big.Int conversion?

Reply:
Good call.  This was originally done before `blockchain` went internal, but now that it is rebased over that, it makes sense to just change the signature.  I'll add a commit for that.

Reply:
After looking at it, I'd rather do it in a separate PR, because it will involve changes in various consumers too.